static/js/tools/map/util.ts

```diff
@@ -344,7 +342,11 @@ export function getRedirectLink(
     parentPlaces: [],
     selectedPlace,
   });
-  return `${MAP_REDIRECT_PREFIX}#${encodeURIComponent(hash)}`;
+  let args = "";
+  if (displayOptions.allowLeaflet) {
+    args += `?${ALLOW_LEAFLET_URL_ARG}=1`;
+  }
+  return `${MAP_REDIRECT_PREFIX}${args}#${encodeURIComponent(hash)}`;
```

Comment:
please add a manual comment equivalent to IFTT to update both files when either is edited

Reply:
done